packages/ui/src/views/repo/repo-summary/components/edit-repo-details-dialog.tsx

```diff
@@ -0,0 +1,90 @@
+import { ChangeEvent, useState } from 'react'
+
+import {
+  AlertDialog,
+  AlertDialogContent,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  Button,
+  ButtonGroup,
+  Icon,
+  Text,
+  Textarea
+} from '@/components'
+
+interface EditRepoDetailsDialog {
+  showEditRepoDetails: boolean
+  description?: string
+  onSave: (desc: string) => void
+  onClose: () => void
+  updateRepoError?: string
+  isSubmitting: boolean
+  isSubmitted: boolean
+}
+
+export const EditRepoDetails = ({
+  showEditRepoDetails,
+  description,
+  onSave,
+  onClose,
+  updateRepoError,
+  isSubmitting,
+  isSubmitted
+}: EditRepoDetailsDialog) => {
+  const [newDesc, setNewDesc] = useState<string>(description || '')
+  return (
+    <AlertDialog open={showEditRepoDetails} onOpenChange={onClose}>
+      <AlertDialogContent className="h-80 max-h-[70vh] max-w-[460px]" onOverlayClick={onClose}>
+        <AlertDialogHeader>
+          <AlertDialogTitle className="mb-4">Repository Description</AlertDialogTitle>
+        </AlertDialogHeader>
+        <Text className="inline-block leading-none text-foreground-7" size={1}>
```

Comment:
```suggestion
        <Text className="inline-block leading-none text-foreground-2" size={1}>
```

<img width="726" alt="image" src="https://github.com/user-attachments/assets/b213bbe5-a082-496e-8b04-76db7657eef7">